plugins/@grouparoo/mongo/__tests__/table-import/get-changed-rows-count.ts

```diff
@@ -0,0 +1,31 @@
+import "@grouparoo/spec-helper";
+
+import { getConfig, usersCollectionName } from "../utils/data";
+import { getChangedRowCount } from "../../src/lib/table-import/getChangedRowCount";
+import { connect } from "../../src/lib/connect";
+import { FilterOperation } from "@grouparoo/app-templates/dist/source/table";
+
+// these used and set by test
+const { appId, appOptions } = getConfig();
+
+async function callGetChangedRowsCount() {
+  const connection = await connect({ appOptions, app: null, appId: null });
```

Comment:
need to call `await client.close();`?

Reply:
it should probably be in a `beforeAll` and `afterAll` in all these tests. Something like
`beforeAll`: `connection = await connect({appOptions})`
`afterAll`: `await disconnect({connection})`